docs/Configuration.md

```diff
@@ -2267,3 +2267,21 @@ This option allows comments in `package.json`. Include the comment text as the v
   }
 }
 ```
+
+### `workerIdleMemoryLimit` \[number]
+
+Specifies the memory limit for workers before they are recycled and is primarily a work-around for [this issue](https://github.com/facebook/jest/issues/11956);
+
+After the worker has executed a test the memory usage of it is checked. If it exceeds the value specified the worker is killed and restarted. The limit can be specified in 2 ways
+
+* < 1 - The value is assumed to be a percentage of system memory. So 0.5 sets the memory limit of the worker to half of the total system memory
+* > 1 - Assumed to be a fixed byte value
+
+```js tab
+/** @type {import('jest').Config} */
+const config = {
+  workerIdleMemoryLimit: 0.2,
+};
+
+module.exports = config;
+```
```

Comment:
Could you add under `js tab` code block, please:

```
    ```ts tab
    import type {Config} from 'jest';

    const config: Config = {
      workerIdleMemoryLimit: 0.2,
    };

    export default config;
    ```
```

This will tell [`docusaurus-remark-plugin-tab-blocks`](https://github.com/mrazauskas/docusaurus-remark-plugin-tab-blocks) plugin to create JS and TS tabs. You can see them in deploy preview or: https://jestjs.io/docs/next/configuration

Reply:
No problem